tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -279,55 +267,3 @@ def transform_to_months(quarter):
 def month_to_int(month):
     """Return the integer value of a month."""
     return datetime.strptime(month, '%b').strftime('%m')
-
-
-def case_aggregates_by_month(df, dfs_status):
-    """Return case aggregates by month."""
-    section = str(df.section)  # section -> text
```

Comment:
the remaining changes were made to resolve circular import errors from using `value_is_empty` within the validators file